db/column_family.cc

```diff
@@ -443,26 +443,17 @@ bool SuperVersion::Unref() {
   return previous_refs == 1;
 }
 
-void SuperVersion::Cleanup(const bool noImmMemoryContribution) {
+void SuperVersion::Cleanup() {
   assert(refs.load(std::memory_order_relaxed) == 0);
   // Since this SuperVersion object is being deleted,
   // decrement reference to the immutable MemtableList
   // this SV object was pointing to.
   imm->Unref(&to_delete);
   MemTable* m = mem->Unref();
   if (m != nullptr) {
-    // Typically, if the m memtable was not made
-    // immutable, and therefore was not added to the
-    // imm list, it does not contribute to the imm
-    // memory footprint (and actually is not part of
-    // the 'imm' MemtableList at all).
-    // At the moment, noImmMemoryContribution is only
-    // used by the experimental 'MemPurge' prototype.
-    if (!noImmMemoryContribution) {
```

Comment:
Remove changes made during #8454 when the mempurge process was happening in the middle of the SwitchMemtable process.